packages/IntegrationTest/tests/SampleTests.tsx

```diff
@@ -13,21 +13,21 @@ import {functionTest, componentTest} from './lib/TestDefinition';
 /**
  * An example of a passing test
  */
-functionTest('SampleFunctionTest', () => {
+functionTest('NativeModuleExample', () => {
   assert.equal(Platform.OS, 'windows', 'Platform.OS should be windows');
 });
 
 /**
  * An example of a failing test using ".skip" not to run
  */
-functionTest.skip('SampleFailingTest', () => {
+functionTest.skip('FailingExample', () => {
   assert.ok(Platform.Version, 'Platform should define a version');
```

Comment:
>Version [](start = 21, length = 7)

should this just be `false`?  The code now reads that it woud pass.. I realize I missed this the previous time I reviewed this fie :)

Reply:
Sounds good. That will also make more sense if we ever fix the PlatformConstants bug 🙂.